src/routes/AuthLayoutRouter/AuthLayoutRouter.js

```diff
@@ -76,7 +76,8 @@ export default function AuthLayoutRouter (props) {
       '/groups/:joinGroupSlug/join/:accessCode',
       '/:context(groups)/:groupSlug/:view(events|groups|map|members|projects|settings|stream|topics|custom)?',
       '/:context(all|public)/:view(events|groups|map|members|projects|settings|stream|topics)?',
-      '/:context(all|welcome)'
+      '/:context(all|welcome)',
```

Comment:
you could just make this '/:context(all|welcome|my)',

Reply:
wait, what is welcome??

Reply:
I didn't add it; assuming welcome is either the new user welcome UI.

Maybe adding `my` will work but then we'd have to add the `?` as well and I didn't test whether that would screw up the other routing

Reply:
the | is an or though, so you wouldn't need a ? here